CITATION.cff

```diff
@@ -0,0 +1,57 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+
+# Project information
+authors:
+  - name: "National Optical Astronomy Observatories"
+  - name: "The IRAF community"
+title: IRAF
+type: software
+abstract: "Image Reduction and Analysis Facility "
+repository-code: "https://github.com/iraf-community/iraf"
+url: "https://iraf-community.github.io"
+license-url: "https://github.com/iraf-community/iraf/blob/main/COPYRIGHT"
+
+# Version information
+version: "2.17"
+doi: "10.5281/zenodo.5816744"
```

Comment:
Please make this information rather an identifier in the "Project information" section above since further identifiers might be added in the future.  The type would be `type: doi`.